### ISSUE TITLE
Add SemanticsRole.tooltip to RawTooltip

### ISSUE THREAD

**Issue by victorsanni** [flutter/flutter#180320]
### Use case

After https://github.com/flutter/flutter/pull/177678 adds RawTooltip, the widget should have SemanticsRole.

### Proposal

Add SemanticsRole.tooltip to the new RawTooltip.
